tests/fullstack-test/mpp/remote_read_virtual_column.test

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Preparation.
+=> DBGInvoke __init_fail_point()
+
+mysql> drop table if exists test.test_vir;
```

Comment:
Can you add some test cases about partition tables ?

Reply:
I found a bug of partition table when virtual column is used, so we cannot recovery this bug on master for partition table. (but we can recovery on 7.5.5)